recipes-core/base-files/base-files/system-backup.txt

```diff
@@ -8,6 +8,7 @@
 /etc/systemd/network/
 /etc/ofdpa-grpc.conf
 -/etc/modprobe.d/i2c-ismt.conf
+-/etc/modprobe.d/i2c-i801.conf
```

Comment:
```suggestion
-/etc/modprobe.d/i2c-i801.conf
-/etc/modprobe.d/i2c-ismt.conf
```

alphabetical order plz kthxbye (I know the rest isn't, but lets try to keep new stuff ordered)

Reply:
Done.